TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -8,8 +8,7 @@ public class TypewiseAlertTest
     [Fact]
     public void InfersBreachAsPerLimits()
     {
-      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+       Assert.True(TypewiseAlert.InferBreach(12, 20, 30) == TypewiseAlert.BreachType.TOO_LOW);
```

Comment:
Test Coverage is in complete ....add test case for email notification

Reply:
Added Testcases for the Fake Notification classes